.github/workflows/create-release-on-branch-changes.yml

```diff
@@ -54,7 +54,7 @@ jobs:
         with:
           command: yarn install --frozen-lockfile
           attempt_limit: 3
-          attempt_delay: 2000
```

Comment:
As I read it this will retry 3 times with a 10 minutes separation.

10 minutes is a long time to way. How about changing it to 15 (or 20 or 30) times with 1 minute separation?

Reply:
Not a bad idea. Let's try that.